src/modules/cpu/kernel/glitch.hpp

```diff
@@ -0,0 +1,1133 @@
+#include <x86intrin.h>
+#include <smmintrin.h>
+#include <immintrin.h>
+#include <fstream>
+#include <cstring>
+#include "rppdefs.h"
+#include "rpp_cpu_simd.hpp"
+#include "rpp_cpu_common.hpp"
+
+template <typename T> void convert_f32_Pln3_Pkd3(T *srcPtrChannel, T *dstPtrChannel, RpptDescPtr srcDescPtr, RpptDescPtr dstDescPtr, int width)
```

Comment:
Please rename the function to convert_f32_pln3_pkd3
Please move template <typename T> to previous line
Applicable to all such instances in this file